packages/stark-ui/src/modules/input-mask-directives/directives/text-mask.directive.ts

```diff
@@ -0,0 +1,112 @@
+import { Directive, ElementRef, forwardRef, Inject, Input, OnChanges, Optional, Provider, Renderer2, SimpleChanges } from "@angular/core";
+import { COMPOSITION_BUFFER_MODE, NG_VALUE_ACCESSOR } from "@angular/forms";
+import { MaskedInputDirective, TextMaskConfig as Ng2TextMaskConfig } from "angular2-text-mask";
```

Comment:
Why importing `TextMaskConfig` as `Ng2TextMaskConfig` ?

Reply:
Because the `MaskedInputDirective` from `angular2-text-mask` also defines the typings for the config as TextMaskConfig which is the same name we have in our own typings. So we need to give a different name otherwise Typscript gives a compilation error.

Reply:
Okok. Thanks for the explanation 😊